frontend/src/components/placementVisuals/Placement.tsx

```diff
@@ -31,6 +37,57 @@ function Placement({ stepID, sampleIDs }: PlacementProps) {
     const destinationContainers = labworkStepPlacement.destinationContainers
     const activeSourceContainer = labworkStepPlacement.activeSourceContainer
     const activeDestinationContainer = labworkStepPlacement.activeDestinationContainer
+    const step = selectStepsByID(store.getState())[stepID]
+    const usesSamplesheet = step.name === EXPERIMENT_RUN_ILLUMINA_STEP
+    const cells = useAppSelector((state) =>  activeDestinationContainer?.name && selectContainer(state)({ name: activeDestinationContainer.name })?.cells)
```

Comment:
I would prefer this ;)
```ts
const cells = useAppSelector((state) =>  activeDestinationContainer?.name !== undefined ? selectContainer(state)({ name: activeDestinationContainer.name })?.cells : undefined)
```

Typescript would otherwise think that cells can also be an empty string.

Reply:
Mmm i think you actually use null as a normal value somewhere too ... ok.
Typescript can go to the mall ... and shop for empty strings...

Reply:
I can't get Bing to generate typescript shopping at a mall for empty strings...
![image](https://github.com/user-attachments/assets/71789502-614e-42e0-b046-3f0ae04bfcbb)

Reply:
Bing sure does have a lot of strings...

Reply:
> Mmm i think you actually use null as a normal value somewhere too ... ok. Typescript can go to the mall ... and shop for empty strings...

I do. It's to represent tubes without parent.